src/main/java/org/gridsuite/explore/server/services/ExploreService.java

```diff
@@ -277,4 +277,28 @@ public void assertCanCreateCase(String userId) {
             }
         }
     }
+
+    public void updateElement(UUID id, ElementAttributes elementAttributes, String userId) {
+        directoryService.updateElement(id, elementAttributes, userId);
+        ElementAttributes elementsInfos = directoryService.getElementInfos(id);
+        // send notification if the study name was updated
+        if (STUDY.equals(elementsInfos.getType()) && StringUtils.isNotBlank(elementAttributes.getElementName())) {
```

Comment:
We already check the type in `notifyStudyUpdate` and why do we need to check `isNotBlank` for the name ?